engine/src/main/resources/application.conf

```diff
@@ -32,6 +32,9 @@ spray.can {
 system {
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  max-retries = 3
```

Comment:
The engine-level max should probably be higher than this.  For JES I believe we usually allow 3 preemption attempts, which means the engine would be asked to kick off a fourth attempt (which the JES backend would run non-preemptibly).